### DISCUSSION TITLE
France Musique Radio

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#240] [Feature Requests]
Is there a way to add these stations to rAudio-1?

https://www.francemusique.fr/

**Comment by rern** [comment]
https://icecast.radiofrance.fr/francemusique-hifi.aac?id=radiofrance

**Reply by R101G** [reply]
Thanks rern, but it is the themed streams I am after. They seem to be m3u, but I cannot find a URL anywhere.

**Reply by rern** [reply]
Try:
```sh
curl -L https://github.com/rern/rAudio-addons/raw/main/webradio/radiofrance.tar.xz | bsdtar xvf - -C /
chown -R http:http /srv/http/data/webradios*
count=$( ls -1 /srv/http/data/webradios | wc -l )
sed -i 's/\("webradio": \).*/\1'$count'/' /srv/http/data/mpd/counts
```

I think we should add them to default stations in next release but they fall short of missing metadata.

**Reply by R101G** [reply]
Thanks again rern. It works perfectly but, as you say, the lack of metadata is disappointing. The BBC has the same problem.

**Reply by klassikmann** [reply]
@rern 
Very smart. Thank you.
To complete the streaming offer of Radio France I recommend to include the AAC streams of FIP (formely known as "France Inter Paris"). Pls look here: https://github.com/JV-conseil-Internet-Consulting/Radio-France-Flux-HD-AAC

**Reply by rern** [reply]
@klassikmann,

Done.
Run the commands again.

**Comment by R101G** [comment]
rern,

Having spent some time listening to the France Musique streams now, I am not at all sure that the sound quality is good enough for them to be included in the image. Perhaps an add-on might be an option?

**Reply by klassikmann** [reply]
@R101-Github 
I didn't perform listening tests plus static stream analysis using the Radio France streams up to now. But I will do that the time coming, in particular the France Musique streams. What I can say now: The streams are 192kb/48kHz AAC LE and therefore not comparable to the BBC Radio streams 320kb/48kHz (The HD streams of BBC Radio are region protected. Outside GB you have access to 128kB MP3 only.). I'm a frequent listener to BBC3, I like it. 

To the topic of inclusion into forthcoming images: 
I agree **not** to include these and future findings into release images to avoid a more and more "fat" list of webradios in my installation. I vote for the radio paradise streams in the image, such that new users find a non-empty webradio list to be used also as a pattern to add entries according personal taste.

Alternatively I recommend to build and published a directory of high-definition webradios where entries can be adopted into a user's private webradio list as an extension to the addons-menu.

**Reply by R101G** [reply]
@klassikmann 
I agree that the RP streams should be included. They sound good, and I believe they have their own source for cover art, which probably means the setup is different.

I have no complaint about the delivery system of the France Musique streams, but high bit rates and AAC do not improve the quality of a badly-mastered source file, and I think that is probably the case here.  :)

**Reply by ceejayemm** [reply]
From my point of view I would prefer any such bundling of radio stations to be as an extra addon. I would add any BBC stations (being in the UK) but not necessarily any other localised stations. Adding to the image would only, as @klassikmann says, make the image 'fat' and I don't want this extra size for something I wouldn't use. The idea of a published directory of high definition radio stations is also I think a good one

Chris
